src/Components/Account.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Parse from "parse";
+
+    const Notifications = () => {
+      try {
+        getCurrentUser()
+      } catch (error) {
+        alert("Error: " + error.code + " " + error.message);
+      }
+    
+    return (
+      <ul>
+        <li>
+          <header className="welcome">
+            <h2>
+              Welcome journalist this is your account. 
+            </h2>
+          </header>
+        </li>
+      </ul>
```

Comment:
Why having a list (ul = unordered list) if you only have 1 list item inside it?
Also, indentation :)